runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpStateSingle.java

```diff
@@ -61,6 +61,8 @@
 	private int startLevelExp = 0;
 	private int endLevelExp = 0;
 
+	private boolean hasGoal = false;
```

Comment:
nit-pick: `false` is the default value, doesn't need to be explicitly specified.
```suggestion
	private boolean hasGoal;
```